navit/vehicle/demo/vehicle_demo.c

```diff
@@ -244,7 +257,8 @@ static void vehicle_demo_timer(struct vehicle_priv *priv) {
                     ci.y = pos.y + dy * len / slen;
                     priv->direction =
                         transform_get_angle_delta(&pos, &c, 0);
-                    priv->speed=priv->config_speed;
+                    priv->speed=priv->config_speed + ((rand() % 5 + 1) * (rand() %2 == 0?-1:1));  // a little random + or - 1 to 5 km/h
+                    priv->height=priv->height + ((rand() % 50 + 1) * (rand() %2 == 0?-1:1));       // a little random + or - 1 to 50 m
```

Comment:
Is it a good idea to have a vehicle which constantly changes its height? In my mind this could introduce problems where navit could recalculate the route because the vehicle is suddenly bigger than the bridge where the vehicle is already under. I don't have anything against the random nature of the height in itself, but chat about keeping this to the init?